src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/LegacyPrisonIncentiveLevelResource.kt

```diff
@@ -11,31 +11,25 @@ import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.incentivesapi.config.ErrorResponse
-import uk.gov.justice.digital.hmpps.incentivesapi.dto.prisonapi.IepLevel
+import uk.gov.justice.digital.hmpps.incentivesapi.dto.LegacyPrisonIncentiveLevel
 import uk.gov.justice.digital.hmpps.incentivesapi.service.PrisonIncentiveLevelAuditedService
-import uk.gov.justice.digital.hmpps.incentivesapi.util.ensure
 
 @RestController
 @RequestMapping("/iep", produces = [MediaType.APPLICATION_JSON_VALUE])
-@Tag(name = "Prison incentive levels", description = "Retrieve incentive levels for a prison")
+@Tag(name = "[Legacy] Prison incentive levels", description = "Incentive levels available in prisons")
 @Deprecated("Use `/incentive/prison-levels/…`")
-class IepLevelsResource(
+class LegacyPrisonIncentiveLevelResource(
```

Comment:
I like the ugly name to really make the point this shouldn't be used, even if this is just an hint for us internally ❤️